src/components/Header.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react";
+import { getUserFolder, getUserId } from "../api.js";
+import "./Header.css";
+export default Header;
+function Header() {
+  return (
+    <header>
+      <nav>
+        <div className="nav-contents">
+          <a href="./folder">
+            <img
+              className="header-logo-img"
+              src="/imgs/logo.png"
+              alt="코드잇로고"
+            ></img>
+          </a>
+          <UserProfileInHeader />
+        </div>
+      </nav>
+      <UserFolderProfile />
+    </header>
+  );
+}
+
+function UserProfileInHeader() {
+  const [user, setUser] = useState(null);
+  const loginImg = <img src="/imgs/로그인.png" alt="로그인"></img>;
+  useEffect(() => {
+    async function handleLoginProfile() {
+      const user = await getUserId();
+      setUser(user);
+    }
+    handleLoginProfile();
+  }, []);
+  return (
+    <a href="./folder" className="user-profile-login">
+      {user ? (
+        <>
+          <img
+            className="user-profile-login-img"
+            src={user.profileImageSource}
+            alt="사용자 이미지"
+          ></img>
+          <span className="user-profile-login-email">{user.email}</span>
+        </>
+      ) : (
+        loginImg
+      )}
+    </a>
+  );
+}
+
+function UserFolderProfile() {
+  const [userFolder, setUserFolder] = useState(null);
+  useEffect(() => {
+    async function handleUserFolderProfile() {
+      const userFolder = await getUserFolder();
+      setUserFolder(userFolder);
+    }
+    handleUserFolderProfile();
+  }, []);
+  return (
+    <div className="userfolder-content">
+      {userFolder ? (
+        <>
+          <img
+            className={"userfolder-content-img"}
+            src={userFolder.folder.owner.profileImageSource}
+            alt="폴더 사용자 이미지"
+          ></img>
+          <span className="userfolder-content-name">
+            {userFolder.folder.owner.name}
+          </span>
+          <span className="userfolder-content-foldername">
+            {userFolder.folder.name}
+          </span>
+        </>
+      ) : (
+        <span></span>
+      )}
```

Comment:
[중요도: 중간] 굳이 빈 span 태그를 넣어주는 삼항연산자보다 && 연산자를 쓰는게 더 나아보이네요~
```suggestion
      {userFolder && (
        <>
          <img
            className={"userfolder-content-img"}
            src={userFolder.folder.owner.profileImageSource}
            alt="폴더 사용자 이미지"
          ></img>
          <span className="userfolder-content-name">
            {userFolder.folder.owner.name}
          </span>
          <span className="userfolder-content-foldername">
            {userFolder.folder.name}
          </span>
        </>
      )}
```

Reply:
https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/pull/165/commits/2e221d0fe074940b7671e41e966c76110ba336c8

&&연산자가 깔끔하게 좋군요!

Reply:
조건부 렌더링에 대해 알아보심 좋습니다.